component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/constants/BackupCodeAuthenticatorConstants.java

```diff
@@ -48,6 +48,12 @@ public class BackupCodeAuthenticatorConstants {
     public static final String IS_INITIAL_FEDERATED_USER_ATTEMPT = "isInitialFederationAttempt";
     public static final String CODE_MISMATCH = "codeMismatch";
     public static final String GET_PROPERTY_FROM_IDENTITY_CONFIG = "getPropertiesFromLocal";
+    public static final String MAX_ATTEMPTS_EXCEEDED = "MAX_ATTEMPTS_EXCEEDED";
+    public static final String ADMIN_INITIATED = "ADMIN_INITIATED";
```

Comment:
since these 2 constants are used only in the test class, no need to have them defined as public constants here

Reply:
These are used in identity apps backup-code.jsp as well.